Dalamud/Game/Text/XivChatType.cs

```diff
@@ -42,12 +43,13 @@ public enum XivChatType : ushort // FIXME: this is a single byte
     /// <summary>
     /// The outgoing tell chat type.
     /// </summary>
+    [XivChatTypeInfo("Tell (Outgoing)", "tell_outgoing", 0xFFFF69B4)] // Do we want this?
     TellOutgoing = 12,
 
     /// <summary>
     /// The incoming tell chat type.
     /// </summary>
-    [XivChatTypeInfo("Tell", "tell", 0xFFFF69B4)]
+    [XivChatTypeInfo("Tell (Incoming)", "tell", 0xFFFF69B4)]
```

Comment:
the slug `tell` also needs to be adjusted

Reply:
Fixed. I also added some more colors.